girder/api/v1/file.py

```diff
@@ -244,6 +244,9 @@ def readChunk(self, params):
                'so you should set it to the index of the byte one past the '
                'final byte you wish to receive.', dataType='integer',
                required=False)
+        .param('contentDisposition', 'Specify the Content-Disposition response '
+               'header disposition-type value', required=False,
```

Comment:
You can specify a ``default`` value here. Even though the API layer just sets it to ``None``, internally it will always default to ``attachment``.